updatecli/updatecli.d/golang.yml

```
@@ -1,4 +1,4 @@
-name: Bump `shared-tools` Golang version
+name: Bump `shared-tools` Golang version to last paker image used one
```

```suggestion
name: Bump `shared-tools` Golang version
```

No need to change the manifest title as it is an implementation choice

---

Alternative: fix the typo

```suggestion
name: Bump `shared-tools` Golang version to the version of the last packer image
```